src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -1,22 +1,160 @@
+import { useRouter } from 'next/router';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { dehydrate } from 'react-query';
+
+import { useGetOrganizerByIdQuery } from '@/hooks/api/organizers';
+import {
+  OwnershipRequest,
+  RequestState,
+  useGetOwnershipRequestsQuery,
+} from '@/hooks/api/ownerships';
+import { Alert, AlertVariants } from '@/ui/Alert';
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Icon } from '@/ui/Icon';
+import { Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
 import { Page } from '@/ui/Page';
+import { Stack } from '@/ui/Stack';
+import { Title } from '@/ui/Title';
 import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
 
+import { OwnershipsTable } from './OwnershipsTable';
+
 const Ownership = () => {
+  const router = useRouter();
+  const { t, i18n } = useTranslation();
+
+  const organizerId = useMemo(
+    () => router.query.organizerId as string,
+    [router.query.organizerId],
+  );
+  const getOrganizerByIdQuery = useGetOrganizerByIdQuery({
+    id: organizerId,
+  });
+  // @ts-expect-error
+  const organizer = getOrganizerByIdQuery?.data;
```

Comment:
I had forgotten how much it bothered me that our queries lost the types of the things :D Can we maybe still do `organizer: Organizer =` for extra safety